packages/constants/src/network/network.constant.ts

```diff
@@ -6,35 +6,51 @@ export enum Network {
   CUSTOM = 'Custom'
 }
 
-export enum NetworkServer {
-  MAINNET = 'wss://xrplcluster.com',
-  TESTNET = 'wss://s.altnet.rippletest.net:51233',
-  DEVNET = 'wss://s.devnet.rippletest.net:51233',
-  AMM_DEVNET = 'wss://amm.devnet.rippletest.net:51233',
-  MAINNET_S1 = 'wss://s1.ripple.com', // Clio
-  MAINNET_S2 = 'wss://s2.ripple.com' // Clio
+export const MAINNET_CLIO_NODES = ['wss://s1.ripple.com', 'wss://s2.ripple.com'];
+export const MAINNET_NODES = ['wss://xrplcluster.com', ...MAINNET_CLIO_NODES];
+export const TESTNET_NODES = ['wss://s.altnet.rippletest.net:51233', 'wss://testnet.xrpl-labs.com'];
+export const DEVNET_NODES = ['wss://s.devnet.rippletest.net:51233'];
+export const AMM_DEVNET_NODES = ['wss://amm.devnet.rippletest.net:51233'];
+
```

Comment:
We can add 
Hooks-Testnet

wss://hooks-testnet-v3.xrpl-labs.com

Since it's listed on the XRPL servers page

Reply:
Let's add this in another MR :) The scope of this one will be only about improving the connection